packages/@lwc/module-resolver/src/utils.ts

```diff
@@ -68,17 +68,17 @@ export function getModuleEntry(
 
 export function normalizeConfig(
     config: Partial<ModuleResolverConfig>,
-    scope: string
+    scope?: string
```

Comment:
`normalizeConfig` is only used internally in `resolveModule`. No need to make this parameter optional, let's remove it.